### ISSUE TITLE
Feature: Convert to pre-defined file size

### ISSUE THREAD

**Issue by TriVoxel** [gianni-rosato/aviator#52]
Instead of just CRF, would be nice to have an option to convert to a specific file size. Math can be done to determine the bitrate needed for a specific output file size. This would be useful for something like sharing screencasts on Discord, where the maximum file size limit is 8 MB, or to other social media sites/CDNs. [Here's a guide](https://stackoverflow.com/questions/29082422/) that works for X264. I suspect we could do something similar for AV1, with better results.

Basically, UI is something like this:

- User selects from CRF to "File size" option.
- There is a text box for size, to the right a dropdown to choose Bit, Byte, MB, or GB (or we parse entry for pattern like MB, MiB, GB, GiB, etc.)
- User inputs size
- Program calculates the bitrate and outputs to the exact file size...

Benefits:

- Consistently get desired size
- Good for apps that have a hard limit for file size
- Convenient & easy

**Comment by gianni-rosato** [comment]
This is much easier said than done. If you're interested in targeting a particular size, see Autocompressor: https://autocompressor.net/ - if I implemented such a feature, I'd want to be able to do it with the polish that this tool has, and this is currently not something I'm capable of doing. I'm going to close this one as not planned because it doesn't necessarily align with the project goals
